metadata.yaml

```diff
@@ -2,9 +2,11 @@ homepage: "https://www.redditinc.com/advertising"
 documentation: "https://advertising.reddithelp.com/en/categories/measurement/install-reddit-pixel#googleTM"
 versions:
   # Latest Version
+  - sha: 19629745821b7140315cf4c429625a91351b8b48
+    changeNotes: This release includes support to pass a unique conversion ID that corresponds to a distinct conversion event. Conversion ID is used for deduplication and prevents the same conversion event from being processed more than once if it is sent multiple times.
```

Comment:
Change notes uses the rough same copy as the Help Text for the new Conversion ID field which was previously approved by Katie Gold.

Reply:
Nit: "de-duplication" should be hyphenated.